src/Univapay/Enums/OnlineBrand.php

```diff
@@ -13,12 +13,19 @@ public static function WE_CHAT_ONLINE() { return self::create(); }
     public static function ALIPAY_CHINA() { return self::create(); }
     public static function ALIPAY_HK() { return self::create(); }
     public static function B_KASH() { return self::create(); }
+    public static function BOOST() { return self::create(); }
     public static function BPI() { return self::create(); }
     public static function DANA() { return self::create(); }
     public static function EASYPAISA() { return self::create(); }
     public static function GCASH() { return self::create(); }
+    public static function GRAB_SG() { return self::create(); }
     public static function KAKAOPAY() { return self::create(); }
+    public static function KREDIVO_ID() { return self::create(); }
+    public static function MAYA() { return self::create(); }
+    public static function NAVER_PAY() { return self::create(); }
```

Comment:
is the underscore `_` correct in `NAVER_PAY` as it is `NAVERPAY` in GPP API and the NewWallets20231115 xlsx document ?

Reply:
Yes, because based on the API code for OnlineBrand, the entry name there is not overridden to be `naverpay`.